### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by ummyhajj** [MDAnalysis/mdanalysis#4495] [Installation] [answered]
I would like to be guided on how to install the MDAnalysis library and to start contributing on projects.

**Comment by hmacdope** [comment, marked as answer]
@ummyhajj check out our userguide which should help you get started: https://userguide.mdanalysis.org/stable/index.html

**Reply by Dehbeie** [reply]
Thank you, just wanted to ask same question.

**Reply by ummyhajj** [reply]
Thank you very much for the link. It was helpful 👌😇
